Shared/mods/deathmatch/logic/luadefs/CLuaXMLDefs.cpp

```diff
@@ -184,6 +185,11 @@ int CLuaXMLDefs::xmlLoadFile ( lua_State* luaVM )
                             }
                         }
 
+                        xmlFile->GetLastError ( strError );
```

Comment:
`strError.empty()` could return **true**, you might want to check for it in this case.